actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -176,7 +184,8 @@ def fragment_name_with_digest(name) #:nodoc:
 
       # TODO: Create an object that has caching read/write on it
       def fragment_for(name = {}, options = nil, &block) #:nodoc:
-        read_fragment_for(name, options) || write_fragment_for(name, options, &block)
+        force = options.is_a?(Hash) && options[:force]
```

Comment:
Can you change this to use keyword arguments? I'd also like to see the conditional expanded.

```ruby
def fragment_for(name = {}, force: false, **options, &block)
  unless force
    read_fragment_for(name, **options) || write_fragment_for(name, **options, &block)
  end
end
```